pkg/function/server/server.go

```diff
@@ -90,39 +90,53 @@ func (s *server) Start(ctx context.Context, inputOptions ...Option) {
 		inputOption(opts)
 	}
 
-	cleanup := func() {
-		if _, err := os.Stat(opts.sockAddr); err == nil {
-			if err := os.RemoveAll(opts.sockAddr); err != nil {
-				log.Fatal(err)
-			}
+	cleanup := func() error {
+		_, err := os.Stat(opts.sockAddr)
+		if err == nil {
+			err = os.RemoveAll(opts.sockAddr)
 		}
+		return err
+	}
```

Comment:
It seems the logic within cleanup changed a bit. Before the change, if `os.Stat(opts.sockAddr)` returns non-nil error, we ignore it. After the change, we return the error. @vigith @whynowy thoughts?

Reply:
yes, file not found should be ignored.

Reply:
According to Vigith's reply, suggesting the following change:

```suggestion
	cleanup := func() error {
		if _, err := os.Stat(opts.sockAddr); err == nil {
			return os.RemoveAll(opts.sockAddr)
		}
		return nil
	}
```